src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -649,5 +864,14 @@ Microsoft::WRL::ComPtr<IDWriteTextFormat> DxFontRenderData::_BuildTextFormat(con
                                                      _fontSize,
                                                      localeName.data(),
                                                      &format));
+
+    // If the OS supports IDWriteTextFormat3, set the font axes
+    ::Microsoft::WRL::ComPtr<IDWriteTextFormat3> format3;
+    if (!FAILED(format->QueryInterface(IID_PPV_ARGS(&format3))) && !_axesVector.empty())

```

Comment:
Maybe make `!_axesVector.empty()` the first condition for shortcut? The `!FAILED(format->QueryInterface` will always be true on supported platforms.

Reply:
good call! changed